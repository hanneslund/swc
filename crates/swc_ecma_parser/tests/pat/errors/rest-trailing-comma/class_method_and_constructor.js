class c {
  constructor(...a,){}
  fn(...a,) {}
}
