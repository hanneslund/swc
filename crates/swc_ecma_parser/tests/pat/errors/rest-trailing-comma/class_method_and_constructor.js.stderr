
  x Trailing comma isn't permitted after a rest element
   ,-[$DIR/tests/pat/errors/rest-trailing-comma/class_method_and_constructor.js:2:3]
 2 | constructor(...a,){}
   :                 ^
   `----

  x Trailing comma isn't permitted after a rest element
   ,-[$DIR/tests/pat/errors/rest-trailing-comma/class_method_and_constructor.js:3:3]
 3 | fn(...a,) {}
   :        ^
   `----
